cves/kernel/CVE-2019-19070.yml

```diff
@@ -19,26 +19,26 @@ curated_instructions: |
   This will enable additional editorial checks on this file to make sure you
   fill everything out properly. If you are a student, we cannot accept your work
   as finished unless curated is properly updated.
-curation_level: 0
+curation_level: 2
 reported_instructions: |
   What date was the vulnerability reported to the security team? Look at the
   security bulletins and bug reports. It is not necessarily the same day that
   the CVE was created.  Leave blank if no date is given.
 
   Please enter your date in YYYY-MM-DD format.
-reported_date:
+reported_date: 2019-11-20
```

Comment:
Unless I'm understanding incorrectly, does this mean the bug was reported after it was announced to the world? I'm not sure this makes sense.

Reply:
patched

Reply:
fixed the dates